src/commands.js

```diff
@@ -0,0 +1,47 @@
+const { bookshelf,
+  Train,
+  Trains,
+  Passenger,
+  Passengers,
+  Station,
+  Stations,
+  Ticket,
+  Tickets,
+  StationAndTrain,
+  StationAndTrainRecords,
```

Comment:
@jrob8577 The file currently only implements basic fetch functions, but it should eventually use all of those imports when more functions are implemented. (theoretically, since it looks like we're definitely not getting to that point this week).